src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1001,22 +997,26 @@ public Player get(BlockFertilizeEvent event) {
 				}
 			}, 0);
 		}
-		// CraftItemEvent REMIND maybe re-add this when Skript parser is reworked?
-//		EventValues.registerEventValue(CraftItemEvent.class, ItemStack.class, new Getter<ItemStack, CraftItemEvent>() {
-//			@Override
-//			@Nullable
-//			public ItemStack get(final CraftItemEvent e) {
-//				return e.getRecipe().getResult();
-//			}
-//		}, 0);
 		// PrepareItemCraftEvent
 		EventValues.registerEventValue(PrepareItemCraftEvent.class, Slot.class, new Getter<Slot, PrepareItemCraftEvent>() {
 			@Override
-			@Nullable
 			public Slot get(final PrepareItemCraftEvent e) {
 				return new InventorySlot(e.getInventory(), 9);
 			}
 		}, 0);
+		EventValues.registerEventValue(PrepareItemCraftEvent.class, ItemType.class, new Getter<ItemType, PrepareItemCraftEvent>() {
+			@Override
+			public ItemType get(final PrepareItemCraftEvent e) {
+				ItemStack item = e.getInventory().getResult();
+				return new ItemType(item != null ? item : new ItemStack(Material.AIR));
```

Comment:
`ItemType` has a constructor which takes a `Material`. You could avoid the `ItemStack` instantiation by splitting this into an if/else and using that constructor